app/controllers/v0/higher_level_reviews/contestable_issues_controller.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module V0
+  module HigherLevelReviews
+    class ContestableIssuesController < AppealsBaseController
+      def index
+        issues = decision_review_service.get_higher_level_review_contestable_issues(
+          user: current_user, benefit_type: params[:benefit_type]
```

Comment:
do we want to have an "allow list" for `benefit_type`? since we interpolate it later? 🤷‍♀️

Reply:
there's a schema for that :D

but Lighthouse will return a 422 if an invalid benefit_type is used. the service will treat that response like all _bad input_ responses. i felt like that was reasonable. should we treat benefit_type differently?